gravitee-apim-console-webui/src/management/api/notifications/notifications-list/notofication-details/notification-details.component.spec.ts

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { HttpTestingController } from '@angular/common/http/testing';
+import { HarnessLoader } from '@angular/cdk/testing';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { UIRouterModule } from '@uirouter/angular';
+import { MatIconTestingModule } from '@angular/material/icon/testing';
+import { InteractivityChecker } from '@angular/cdk/a11y';
+import { MatCheckboxHarness } from '@angular/material/checkbox/testing';
+import { GioSaveBarHarness } from '@gravitee/ui-particles-angular';
+import { MatInputHarness } from '@angular/material/input/testing';
+
+import { NotificationDetailsComponent } from './notification-details.component';
+import { NotificationDetailsModule } from './notification-details.module';
+
+import { User } from '../../../../../entities/user';
+import { CONSTANTS_TESTING, GioHttpTestingModule } from '../../../../../shared/testing';
+import { GioUiRouterTestingModule } from '../../../../../shared/testing/gio-uirouter-testing-module';
+import { AjsRootScope, CurrentUserService, UIRouterStateParams } from '../../../../../ajs-upgraded-providers';
+import { fakeHooks } from '../../../../../entities/notification/hooks.fixture';
+import { fakeNotificationSettings } from '../../../../../entities/notification/notificationSettings.fixture';
+import { Notifier } from '../../../../../entities/notification/notifier';
+import { fakeNotifier } from '../../../../../entities/notification/notifier.fixture';
+import { NotificationSettings } from '../../../../../entities/notification/notificationSettings';
+import { Hooks } from '../../../../../entities/notification/hooks';
+
+describe('NotificationDetailsComponent', () => {
+  let fixture: ComponentFixture<NotificationDetailsComponent>;
+  const API_ID = 'apiId';
+  const NOTIFICATION_ID = 'f7889b1c-2b4c-435d-889b-1c2b4c235da9';
+  const currentUser = new User();
+  currentUser.userPermissions = ['api-notification-u', 'api-notification-d', 'api-notification-c'];
+  let httpTestingController: HttpTestingController;
+  let loader: HarnessLoader;
+
+  describe('notification settings test', () => {
```

Comment:
Is this describe lvl necessary ? or we can remove it ?

Reply:
Done